microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -190,16 +190,29 @@ You can access Collections from an InPrivate window when using a signed-out prof
 <!-- ====================================================================== -->
 ## Compose
```

Comment:
need to have some general text directly in this h2 section.  what can we say about Compose tool/feature in general, that's not specific to the detailed Sidebar or Rewrite h4 sections?  @majenny

Reply:
```
## Compose
The Compose tool helps you draft text within the Microsoft Edge Sidebar, or rewrite selected text on any webpage.

#### Use Compose to draft text within the Edge Sidebar
In the Copilot pane, the Compose tool uses AI to write for you, starting from a short prompt and **Tone**, **Format**, and **Length** selections: ...

#### Use Compose to rewrite selected text on any webpage
The Compose tool can help rephrase what you've written, and iterate using the same **Tone**, **Format**, and **Length** selections. ...
```